code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -34,6 +34,26 @@
 		M.hallucination += 5
 	..()
 
+/datum/reagent/drug/cannabis
+	name = "Cannabis"
```

Comment:
```suggestion
	name = "Cannabis-derived cannabinoid mixture"
```

Reply:
It's not technically accurate plus it isn't necessary. There are plenty of reagents that represent a group of chemicals or have vague names. Drinks, bath salts or space drugs for example.